app/helpers/notify_view_helper.rb

```diff
@@ -79,6 +84,11 @@ def unsubscribe_link(subscription)
     notify_link(url, t(".unsubscribe_link_text"))
   end
 
+  def view_applications_for_link(vacancy)
```

Comment:
We probably want to stick some tracking stuff in the url - see how the other urls in this helper use `**utm_params`. @stevenleggdfe can use this to track how many views to the page come from emails.

Reply:
Yes please! @josephhull676 if you follow the same scheme as the other emails use, email clickthrough rates will then be tracked automatically on the relevant dashboard.